lib/common/theme.dart

```diff
@@ -456,6 +457,29 @@ extension CustomSliderTheme on SliderThemeData {
           .copyWith(color: ProvenanceThemeData._neutral800));
 }
 
+class ValidatorStatusColorScheme {
```

Comment:
Since this doesn't extend `ColorScheme` and is specific to validators whereas `theme.dart` is very generic it seems like `colorForStatus` might fit better in a `/lib/util/validator_util.dart` file as a top level static method named something like `validatorColorForStatus` or similar.